pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -178,7 +178,7 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			klog.V(4).Infof("The attributes of queue <%s> in proportion: deserved <%v>, allocate <%v>, request <%v>, share <%0.2f>",
 				attr.name, attr.deserved, attr.allocated, attr.request, attr.share)
 
-			increased, decreased := attr.deserved.Diff(oldDeserved)
+			increased, decreased := attr.deserved.Diff(oldDeserved, api.Zero)
 			increasedDeserved.Add(increased)
 			decreasedDeserved.Add(decreased)
```

Comment:
do we need to compare `increasedDeserved` and `increased` here?

Reply:
> do we need to compare `increasedDeserved` and `increased` here?

I've not analysed the logic in proportion here. Just beacause of the input parameters changed, this call should be changed, too.